packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
+   * @param joinMeetingParams
+   */
+  export function joinMeeting(joinMeetingParams: JoinMeetingParams): Promise<string | object> {
+    return new Promise<string | object>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      resolve(
+        sendAndHandleStatusAndReasonWithVersion(
+          getApiVersionTag(ApiVersionNumber.V_2, ApiName.Meeting_JoinMeeting),
+          'meeting.joinMeeting',
+          joinMeetingParams,
+        ),
+      );
+    });
+  }
+
+  /** Join meeting parameters */
+  export interface JoinMeetingParams {
+    /** The meeting url to join. */
+    skypeTeamsMeetingUrl: string;
```

Comment:
Given the name of this, I'm going to assume that this is a publicly documented property from somewhere like the graph. Can you please link to documentation for where this property comes from in the comment?

Reply:
@TrevorJoelHarris, added the public documentation in the comment and made a few changes in parameters accordingly.